### PR TITLE
[TESTS splattributes] failing tests to repro bug

### PR DESCRIPTION
**Pull request by gossi**
Repro fro #17147

### CODE REVIEW

**Comment by rwjblue** [comment]
@gossi - the tests passed here, the only failure is with linting

**Comment by rwjblue** [comment]
```
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/glimmer/tests/integration/components/angle-bracket-invocation-test.js
  677:36  error  Replace `'disabled'` with `disabled`  prettier/prettier

```

File: packages/@ember/-internals/glimmer/tests/integration/components/angle-bracket-invocation-test.js [hunk 1]
```diff
@@ -664,6 +664,23 @@ if (EMBER_GLIMMER_ANGLE_BRACKET_INVOCATION) {
...
+        this.registerComponent('foo-bar', {
+          ComponentClass: Component.extend({ tagName: '' }),
+          template: '<button data-bar ...attributes>hello</button>',
+        });
+
+        this.render('<FooBar disabled/>');
+
+        this.assertElement(this.firstChild, {
+          tagName: 'button',
+          attrs: { 'data-bar': '', 'disabled': '' },
```
Comment (rwjblue) [inline packages/@ember/-internals/glimmer/tests/integration/components/angle-bracket-invocation-test.js]:
```suggestion
          attrs: { 'data-bar': '', disabled: '' },
```

**Comment by kategengler** [comment]
Related issue is closed.

**Pull request closed without merging**